app/res/layout/activity_crash_warning.xml

```diff
@@ -0,0 +1,67 @@
+<ScrollView
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="fill_parent"
+    android:layout_height="fill_parent">
+
+    <RelativeLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:paddingBottom="@dimen/content_start"
+        android:paddingLeft="@dimen/login_box_margins"
```

Comment:
I think we're trying to move to following the Android style guidelines rule that all padding/spacing should be multiples of 8dp. We should definitely do a pass at some point to remove existing dimensions in our dimens.xml file that violate that rule, so that we stop trying to use them.